### ISSUE TITLE
Aggressive.cpp

### ISSUE THREAD

**Issue by Taose** [Taose/Vegastrike-taose#86]
Needs fixing urgently.

```
In file included from /home/eyer/Testing/Vegastrike-taose/engine/src/cmd/ai/aggressive.cpp:25:0:
/home/eyer/Testing/Vegastrike-taose/engine/src/python/python_compile.h: In member function ‘PyObject* PythonBasicType::NewObject() const’:
/home/eyer/Testing/Vegastrike-taose/engine/src/python/python_compile.h:43:57: error: ‘PyString_FromString’ was not declared in this scope
             return PyString_FromString( objects.c_str() );
                                                         ^
CMakeFiles/engine_com.dir/build.make:62: recipe for target 'CMakeFiles/engine_com.dir/src/cmd/ai/aggressive.o' failed
make[2]: *** [CMakeFiles/engine_com.dir/src/cmd/ai/aggressive.o] Error 1
CMakeFiles/Makefile2:262: recipe for target 'CMakeFiles/engine_com.dir/all' failed
make[1]: *** [CMakeFiles/engine_com.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by Taose** [comment]
The first thing I'll do is try to confirm this on my system. 
If that passes then I'll try on the Fedora 26.

**Comment by Taose** [comment]
This is what I've done so far. 
Testing System: Ubuntu 16.10

Navigated to my Vegastrike clone and checkout master, headed into the build directory and cleared the pre-existing stuff. 

I then ran cmake, which worked perfectly. 

_note_: We should probably take warnings out the "master" cmake build and only show errors. Leaving warnings and stuff for experimental versions and debugging. 

**Compilation Successful**

Which is bad news because now I have to test it on the Fedora VM.

**Comment by Taose** [comment]
After further research, it appears that this problem is due to Fedora having Python3 installed by default where Python2.7 may be required. 

Edit:

Fedora does not have Python3 installed by default, it has Python2.7.13 by default. Therefore this issue has been downgraded in importance.

**Comment by Taose** [comment]
Additionally, the cmake seems to grab the highest version of python available and not necessarily 2.7

edit: 

A suggested change to cmakelists.txt to prefer 2.7 would be advantageous at present as in [this](http://stackoverflow.com/a/34580995/4647075) answer.

**Comment by Taose** [comment]
Appears to be resolved.
